File: TradeItIosEmsApi/TradeItBrokerListRequest.m
#import "TradeItBrokerListRequest.h"

@implementation TradeItBrokerListRequest

- (id)initWithApiKey:(NSString *)apiKey
     userCountryCode:(NSString * _Nullable)countryCode {
    self = [super init];

    if (self) {
        self.apiKey = apiKey;
        self.countryCode = countryCode;
    }

    return self;
}

@end

File: TradeItIosEmsApi/TradeItBrokerListRequest.swift
class TradeItBrokerListRequest: TradeItRequest, Codable {
    var apiKey: String
    var countryCode: String?
    
    init(apiKey: String, countryCode: String?) {
        self.apiKey = apiKey
        self.countryCode = countryCode
    }
}
